docker/android.cmake

```diff
@@ -0,0 +1,33 @@
+# toolchain file for android targets, see #1110
+
+set(CMAKE_SYSTEM_NAME Android)
+set(CMAKE_SYSTEM_PROCESSOR "$ENV{CROSS_ANDROID_ARCH}")
+set(CMAKE_ANDROID_STANDALONE_TOOLCHAIN /android-ndk)
+set(CMAKE_ANDROID_API "$ENV{CROSS_ANDROID_SDK}")
```

Comment:
We can't auto-detect the Android API, so we manually set it. CMake will still warn about this, but it's irrelevant since it's manually set.